gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -486,6 +486,15 @@
   public static final String ENCRYPT_KEY_LOC = "encrypt.key.loc";
 
   /**
+   * Proxy Filesystem operation properties.
+   */
+  public static final String SHOULD_FS_PROXY_AS_USER = "should.fs.proxy.as.user";
+  public static final boolean DEFAULT_SHOULD_FS_PROXY_AS_USER = false;
+  public static final String FS_PROXY_AS_USER_NAME = "fs.proxy.as.user.name";
+  public static final String FS_PROXY_AS_USER_TOKEN_FILE = "fs.proxy.as.user.token.file";
+  public static final String SUPER_USER_NAME_TO_PROXY_AS_OTHERS = "super.user.name.to.proxy.as.others";
+
```

Comment:
Can you prefix this all with "writer." since the fs proxying is only happening in the writer.